src/search/json_index/json_search.ml

```diff
@@ -167,11 +167,27 @@ let of_entry ({ Entry.id; doc; kind } as entry) html =
             ("parent_type", `String (Text.of_type parent_type));
           ]
   in
+  let occurrences =
+    match occurrences with
+    | Some occ ->
+        `Object
+          [
+            ("direct", `Float (float_of_int occ.Odoc_occurrences.Table.direct));
+            ("indirect", `Float (float_of_int occ.indirect));
+          ]
+    | None -> `Null
```

Comment:
Nit-picking but it would look more consistent to me if we had 0 for both counters here (see the output in the tests).

Reply:
What about "no 'occurrences' field at all"?
I'm not very keen on choosing a wrong value when it was not possible to compute one because the table was not given...

Reply:
Well, at least if we don't put this field at all it reduces the diff of this PR!

Reply:
With 6205f97, if the occurrence table is not given, the field is not included.
However, if the occurrence table is given, but does not contain the occurrence, a use of 0 is counted.